orchagent/interface_rates.lua

```diff
@@ -0,0 +1,90 @@
+-- KEYS - rif IDs
+-- ARGV[1] - counters db index
+-- ARGV[2] - counters table name
+-- ARGV[3] - poll time interval
+-- return nothing
+
+local asic_db = 1 
+local counters_db = ARGV[1]
+local config_db = 4 
```

Comment:
instead of hard coding should we no pass argument as part of runRedisScript ? Similar how we are passing counter_db id

Reply:
Sure